template/middlewares.template.dart

```diff
@@ -4,6 +4,9 @@ import 'package:shelf_helmet/shelf_helmet.dart';
 /// Dont touch this line because it will be replaced by the build function
 const List<String> hashes = [];
 
+/// Dont touch this line because it will be replaced by the build function
+const bool shouldEnforceCsp = false;
```

Comment:
What do you think about just stating that this is the default for all environments and will be overwritten on build with the value from the used environment.
Then we only call enforceCSP when its set in the environment and overwrite this.